tests/ember-intl/tests/integration/helpers/format-date-time-range/input-is-a-date-test.ts

```diff
@@ -30,7 +30,10 @@ module(
         </div>
       `);
 
-      assert.dom('[data-test-output]').hasText('1/23/2014 – 2/26/2015');
+      assert
+        .dom('[data-test-output]')
+        .hasText(/1\/23\/2014\s–\s1\/26\/2014/)
+        .hasText('1/23/2014 – 1/26/2014');
```

Comment:
These assertions show how end-developers can either use a regular expression or the special character corresponding to `<0x2009>`.